en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -17,7 +17,7 @@ Follow the instructions below to deploy WSO2 API Manager (WSO2 API-M) in a distr
     The following steps describe how to download, install, and configure WSO2 API Manager, with five instances.
     
     1.  Download the [WSO2 API Manager](http://wso2.com/products/api-manager/) in each of the five servers in the cluster for distributed deployment.
-    2.  Unzip the WSO2 API Manager zipped archive, and rename each of those directories respectively as Key Manager, Gateway, Publisher, Store, and Traffic Manager.
+    2.  Unzip the WSO2 API Manager zipped archive, and rename each of those directories respectively as Key Manager, Gateway, Publisher, Dev Portal, and Traffic Manager.
```

Comment:
Shouldn't it be **Developer Portal** for the descriptions?

Reply:
Fixed